aws-lc-fips-sys/include/rust_wrapper.h

```diff
@@ -106,4 +106,12 @@ int ERR_GET_FUNC_RUST(uint32_t packed_error);
 #include "openssl/ssl3.h"
 #endif
 
+// The TLS PRF API is exported in AWS-LC main branch, but in the current FIPS branch, this API is defined in a private header.
+int CRYPTO_tls1_prf(const EVP_MD *digest,
```

Comment:
Bindings need to be regenerated, just hadn't done that yet as I had been waiting for the most recent release to avoid merge conflicts. Otherwise this will just work as normal. Please provide feedback on the TLS PRF API at least and don't block on this in particular.